lms/services/ltia_http.py

```diff
@@ -0,0 +1,72 @@
+import uuid
+from datetime import datetime, timedelta
+
+from lms.services import JWTService
+
+
+class LTIAHTTPService:
+    """Send LTI Advantage requests and return the responses."""
+
+    def __init__(self, lti_registration, jwt_service, http):
+        self._lti_registration = lti_registration
+        self._jwt_service = jwt_service
+        self._http = http
```

Comment:
```suggestion
class LTIAHTTPService(HTTPService):
    """Send LTI Advantage requests and return the responses."""

    def __init__(self, lti_registration, jwt_service):
        self._lti_registration = lti_registration
        self._jwt_service = jwt_service
```

I know it is _forbidden_, but if we are an HTTP service, then we would get the benefit of all the extra `get()`, `post()` convenience methods added to the base class.

Reply:
Not a bad idea but I think it will take a bit of work to adapt the tests (that now use the mocked http_service. I rather revisit it later.

Also have friend that once used inheritance over composition once and this computer exploded...